### ISSUE TITLE
Assert CLI command to honor collect settings onlyCategories

### ISSUE THREAD

**Issue by amalitsky** [GoogleChrome/lighthouse-ci#778]
Currently when `collect` command is using the lighthouse config file with `onlyCategories` set subsequent `assert` **fails** with a list of errors like this one:

```
aria-command-name failure for auditRan assertion
        expected: >=1
           found: 0
      all values: 0, 0
```

I think this might be due to the fact that _skipped_  (as requested and expected) audit artifacts are missing from _lhr_ file(s).

**Describe the solution you'd like**
First option would be to read (or duplicate?) the `collection` lighthouse settings for the `assert` command to be aware of missing audits and silently ignore them.

**Describe alternatives you've considered**
Print out warnings that certain audits are missing even when they are expected (i.e. referenced by `preset: 'lighthouse:recommended'`). But it's not clear if we can consider audit _expected_ just because it is mentioned in `preset`.

If _lhr_ file has a list of audits performed we could use that as a source of truth.

**Additional context**
This is my first time using _Lighthouse CI_ hence I could have done something terribly wrong. If so, please excuse my ignorance and point me to the right direction.

**Environment**
- LHCI version: _0.9.0_
- Lighthouse version: _9.3.0_
- OS: Ubuntu
